src/core/time.d

```diff
@@ -2645,9 +2650,10 @@ extern(C) void _d_initMonoTime()
         {
             foreach(i, typeStr; __traits(allMembers, ClockType))
             {
+                import core.internal.abort : abort;
                 // ensure we are only writing immutable data once
                 if(tps[i] != 0)
-                    assert(0, "_d_initMonoTime should only be called once!");
+                    abort("_d_initMonoTime should only be called once!");
```

Comment:
wait a minute, this is not an environmental issue, it's a programming bug. It should remain an assert.